pom.xml

```diff
@@ -253,11 +245,11 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.]]>
 			<artifactId>jackson-databind</artifactId>
 			<version>2.13.4.1</version>
 		</dependency>
-		<dependency>
+		<!-- <dependency>
```

Comment:
Turns out that we do actually use apache commons lang3 all over the place and this is breaking bbchem/indigo builds because the dependency is only in jchem now.